File: scripts/generate_hash.py
import sys, os, hashlib
pwd = sys.argv[1] if len(sys.argv) > 1 else 'X8n$G7vPq4rZ!2sY'
salt = os.urandom(16).hex()
h = hashlib.pbkdf2_hmac('sha512', pwd.encode(), bytes.fromhex(salt), 100000).hex()
print(salt)
print(h)

File: scripts/generate_hash.js
const crypto = require('crypto');
const pwd = process.argv[2] || 'X8n$G7vPq4rZ!2sY';
const salt = crypto.randomBytes(16).toString('hex');
const hash = crypto.pbkdf2Sync(pwd, salt, 100000, 64, 'sha512').toString('hex');
console.log(salt);
console.log(hash);
